frontend/src/features/recaptcha/useRecaptcha.tsx

```diff
@@ -97,6 +101,14 @@ export const useRecaptcha = ({
 
   const captchaRef = useRef<HTMLDivElement | null>(null)
 
+  // Feature flag to control turnstile captcha rollout
+  // defaults to false
+  // todo: remove after full rollout
+  const enableTurnstileFeatureFlag = useFeatureFlagWithDefaults(
```

Comment:
We might want to include why we want to check for `turnstile` in `turnstile`. It took me awhile to realize that both of these (grecaptcha, cf turnstile) should mutually exclusive.

Reply:
Sure, let me elaborate it better